### PR TITLE
doc: relnotes: Zephyr release notes for NCS 1.4.0

### PR DESCRIPTION
**Pull request by carlescufi**
Since we've gone through a full Zephyr release cycle between 1.3.0 and
1.4.0, point to the official Zephyr 2.4.0 release notes.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -51,405 +51,23 @@ sdk-zephyr
...
 
 For a complete list of |NCS| specific commits, run:
 
 .. code-block:: none
 
-   git log --oneline manifest-rev ^0aaee6dcf5
+   git log --oneline manifest-rev ^7a3b253ced
 
-The following list summarizes the most important changes inherited from upstream Zephyr:
+Changes inherited from upstream Zephyr:
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:68]:
```suggestion

```
Delete this line - not needed.

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -51,405 +51,23 @@ sdk-zephyr
...
-    * Added support for ``only_tags`` in the platform definition.
-    * Disabled returning errors on warnings.
-
-* Other:
-
-  * Implemented ``nanosleep`` in the POSIX subsystem.
-  * Deprecated the Zephyr-specific types in favor of the standard C99 int types.
-  * Removed ``CONFIG_NET_IF_USERSPACE_ACCESS``, because it is no longer needed.
-  * Renamed some attributes in the :c:struct:`device` struct: ``config_info`` to ``config``, ``driver_api`` to ``api``, and ``driver_data`` to ``data``.
+Since |NCS| v1.3.0 is based on Zephyr v2.3.0-rc1, and |NCS| v1.4.0 is based on Zephyr 2.4.0, we recommend referring to  Zephyr's 2.4.0 release notes directly: :ref:`zephyr:zephyr_2.4`.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:70]:
```suggestion
The current |NCS| release is based on Zephyr 2.4.0.
See the :ref:`Zephyr 2.4.0 release notes <zephyr:zephyr_2.4>` for a list of changes.
```

**Review by ru-fu — CHANGES_REQUESTED** [review]
If I understand it correctly, this is the PR for the current state on master, not related to the release.

In the release, the release-notes-latest page will not be included, but the content will be part of the release notes. See PR https://github.com/nrfconnect/sdk-nrf/pull/3127/ for the release notes - they should be updated in a similar way.

So this PR is for updating the release-notes-latest page that will be re-added after the release.

**Comment by carlescufi** [comment]
> In the release, the release-notes-latest page will not be included, but the content will be part of the release notes. See PR #3127 for the release notes - they should be updated in a similar way.

OK, so @greg-fer should cherry-pick this commit into #3127?

**Comment by ru-fu** [comment]
> > In the release, the release-notes-latest page will not be included, but the content will be part of the release notes. See PR #3127 for the release notes - they should be updated in a similar way.
> 
> OK, so @greg-fer should cherry-pick this commit into #3127?

#3127 mainly needs the same changes in a different file (release-notes-1.4.0.rst).
This PR here should probably go in separately, since it's independent of the release.

**Comment by carlescufi** [comment]
> > > In the release, the release-notes-latest page will not be included, but the content will be part of the release notes. See PR #3127 for the release notes - they should be updated in a similar way.
> > 
> > 
> > OK, so @greg-fer should cherry-pick this commit into #3127?
> 
> #3127 mainly needs the same changes in a different file (release-notes-1.4.0.rst).

OK, I will wait for @greg-fer to weigh in here before I push anywhere.

**Comment by greg-fer** [comment]
I've applied changes from this PR into 1.4.0 release notes, in PR #3127 , commit 36b2cbcbc0633baccfbc4d9801acf22078cf0c8c.
This PR should indeed go in separately into master to update the release-notes-latest file.

**Review by greg-fer — APPROVED** [review]

**Review by mbolivar-nordic — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]
Wanted to check the output to be sure it's okay, but the CI stopped.
So approving under the assumption that it would have been green ...

**Comment by ru-fu** [comment]
This is now included in https://github.com/nrfconnect/sdk-nrf/pull/3240

**Comment by tejlmand** [comment]
@carlescufi could this be closed, as 1.4 is already out ?

**Comment by carlescufi** [comment]
> This is now included in #3240

Closing this.

**Pull request closed without merging**